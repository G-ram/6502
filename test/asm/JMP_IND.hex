4c 09 06 a2 40 8e 00 01 00 a9 06 8d 05 01 a9 03
8d 04 01 6c 04 01 
