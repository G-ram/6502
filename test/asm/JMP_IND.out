A: 03
X: 40
Y: 00
P: 00010000
S: 20
PC: 608
100 : 40
101 : 00
